### ISSUE TITLE
Uri used for multiple actions

### ISSUE THREAD

**Issue by jbrauchler** [apiaryio/api-blueprint#336]
Is there a way to make it so that if you have multiple actions with the same uri to document that but still have labels for each action that show up above in the documentation.

Something like this....

```
## /pizzas/{pizzaId}

### Customize a pizza [PUT]
+ details for put

### Delete pizza {DELETE]
+ details for delete
```

**Comment by pksunkara** [comment]
Did you want something like the following?

```
## Pizza [/pizzas/{id}]

+ Parameters
    + id - Id of the pizza

### Customize a Pizza [PUT]
...

### Delete a Pizza [DELETE]
...
```

**Comment by jbrauchler** [comment]
I know how to accomplish that, I was trying to make it so that each action within the api has its own label above it.

**Comment by pksunkara** [comment]
Are you looking for something like this?

```apib
## Customize Pizza [PUT /pizzas/{id}]
...

## Delete Pizza [DELETE /pizzas/{id}]
...
```

**Comment by jbrauchler** [comment]
Yes exactly but I attempted that, and it doesn't work because they have the same uri

**Comment by pksunkara** [comment]
@jbrauchler True, you would get a warning on that because they have the same url endpoint and thus should be merged into a single resource instead of 2 different actions according to good API design.

**Comment by jbrauchler** [comment]
Is there a way to get around that by chance?

**Comment by pksunkara** [comment]
AFAIK, no.

**Comment by jbrauchler** [comment]
Okay Thanks

**Comment by kylef** [comment]
@jbrauchler can you please elaborate on what you mean my a label?

The following example will allow you to add description for each action.

```apib
FORMAT: 1A

# Pizza API

## Pizza [/pizzas/{id}]

+ Parameters
    + id: abc

## Customise Pizza [PUT]

Description for customising a pizza

## Delete Pizza [DELETE]

Description for deleting a pizza
```

Rendered:

![screenshot 2019-01-10 at 18 01 05](https://user-images.githubusercontent.com/44164/50987570-e73a0280-1501-11e9-905b-0972a856b600.png)
